userguide/sharing.rst

```diff
@@ -2697,3 +2541,202 @@ Go back to
+:numref:`Figure %s <create_time_machine_share_fig>`.
+
+#. **Share name:** enter a name for the share that is identifiable but
+   less than 27 characters long. The name cannot contain a period. In
+   this example, the share is named *backup_user1*.
+
+#. Click the button for :guilabel:`Mac OS X (AFP)` and enable the
+   :guilabel:`Time Machine` option.
+
+#. Click the :guilabel:`Ownership` button. If the user already exists
+   on the %brand% system, click the drop-down :guilabel:`User` menu to
+   select their user account.  If the user does not yet exist on the
+   %brand% system, type their name into the :guilabel:`User` field and
+   enable the :guilabel:`Create User` option. If the user will be a
+   member of a group that already exists on the %brand% system, click
+   the drop-down :guilabel:`Group` menu to select the group name. To
+   create a new group to be used by Time Machine users, enter the name
+   in the :guilabel:`Group` field and set the
+   :guilabel:`Create Group` option. Otherwise, enter the same name
+   as the user. In the example shown in
+   :numref:`Figure %s <create_tm_auth_user_fig>`,
+   both a new *user1* user and a new *tm_backups* group will be
+   created. Since a new user is being created, this screen prompts for
+   the user password to be used when accessing the share. It also
+   provides an opportunity to change the default permissions on the
+   share. When finished, click :guilabel:`Return` to return to the
+   screen shown in
+   :numref:`Figure %s <create_time_machine_share_fig>`.
+
+#. Click the :guilabel:`Add` button.
+   **Remember to do this or the share will not be created**.
+   Clicking the :guilabel:`Add` button adds an entry to the
+   :guilabel:`Name` frame with the name that was entered in
+   :guilabel:`Share name`.
+
+
+To configure multiple authenticated or Time Machine shares, repeat for
+each user, giving each user their own :guilabel:`Share name` and
+:guilabel:`Ownership`. When finished, click the :guilabel:`Next`
+button twice, then the :guilabel:`Confirm` button to create the
+shares. The Wizard automatically creates a dataset for each share with
+the correct ownership and starts the AFP service so the shares are
+immediately available. The new shares will appear in
+:menuselection:`Sharing --> Apple (AFP)`.
+
+
+.. _create_time_machine_share_fig:
+
+.. figure:: images/afp7a.png
+
+   Creating a Time Machine Share
+
+
+.. _create_tm_auth_user_fig:
+
+.. figure:: images/afp8.png
+
+   Creating an Authenticated User
+
+
+.. _Configuring Time Machine Backups:
+
+Configuring Time Machine Backups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuring a quota for each Time Machine share helps prevent backups
+from using all available space on the %brand% system. Time Machine waits
+two minutes before creating a full backup. Incremental backups are
+created each hour for the next 24 hours. After that, one backup is
+created each day, each week, and each month. Each of these hourly,
```

Comment:
Please verify the explanation of backups with Andrew or Bonnie.

Reply:
Updated text with feedback from Andrew.